src/test/resources/com/puppycrawl/tools/checkstyle/main/InputMainComplexityOverflow.java

```diff
@@ -8,14 +8,22 @@ public class InputMainComplexityOverflow {
 
   public void provokeNpathIntegerOverflow()
     {
-      if (true) {
-        if (true) {
-          if (true) {
-            if (true) {
-              if (true) {
-                if (true) {
-                  if (true) {
-                    if (true) {
+      if (true)
+        {
+        if (true)
+          {
+          if (true)
+            {
+            if (true)
+              {
+              if (true)
+                {
+                if (true)
+                  {
+                  if (true)
+                    {
+                    if (true)
+                      {
```

Comment:
This is incorrect placement of "{". I have explained the issue in detail in the above comment